src/session.ts

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2016 Google Inc. All Rights Reserved.
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
remove this change.

Reply:
Removed the change. Now onward we need to keep copyright year as it. right ?